slyd/slyd/splash/utils.py

```diff
@@ -116,17 +116,17 @@ def _decode(html, default=None):
 
 
 def lazy_property(func):
```

Comment:
this can be removed in favor of `django.utils.functional.cached_property` which has a cleaner implementation